util.scad

```diff
@@ -90,3 +90,22 @@ module mirrored(vector_set, index = 0) {
     children();
   }
 }
+
+// A shape that can be used for push in parts
+module pin(height, radius) {
+  inner_height = height * 0.875;
+  cap_height = (height - inner_height) / 2;
+  difference() {
+    union() {
+      up(-inner_height/2)
+        cylinder(inner_height, radius, radius, $fn=8);
```

Comment:
$fn might be best unspecified, then you could do `pin($fn=3)`, `pin($fn=6)`, `pin($fn=80)` at your discretion

Reply:
Also, this is another place where specifying argument names would aid clarity ie `cylinder(inner_height, r1=radius, r2=radius)`. ANd in this case, if r1/r2 are the same, you can just say 'r=radius`.